### ISSUE TITLE
Any plan of Monterey preinstalled ?

### ISSUE THREAD

**Issue by evilz** [sickcodes/Docker-OSX#432]
Preinstalled Docker-OSX make work easier and faster to reproduct. Do you plan same than catalina for other version ?

**Comment by sickcodes** [comment]
Yes I am considering Monterey because it is small, might make one right now

**Comment by evilz** [comment]
Just to be sure how did you create this img ?  

 
 ​ARG IMAGE_URL='https://images.sick.codes/mac_hdd_ng_auto.img'

This mean we can use image with preinstalled app ?

**Comment by charliefancelli** [comment]
@sickcodes do you have updates on the Preinstalled Docker-OSX with Monterey ?

**Comment by scm007** [comment]
Any updates? I can't get the non-preinstalled ones to work.

**Comment by sickcodes** [comment]
Ill probably be working on this, this week

**Comment by manang** [comment]
news? thanks

**Comment by charliefancelli** [comment]
@sickcodes can you update us ?

**Comment by sickcodes** [comment]
https://images.sick.codes/mac_hdd_ng_auto_monterey.img

**Comment by gdotdesign** [comment]
Can you tell me what is the sudo password for that image? :)

**Comment by thawri1** [comment]
> https://images.sick.codes/mac_hdd_ng_auto_monterey.img
provide for Ventura too, please.
